tutorials/fundamentals/typeof-array.md

```diff
@@ -0,0 +1,20 @@
+Arrays are a special case when using the `typeof` operator.
```

Comment:
I wouldn't say arrays are a special case, because arrays are technically objects. I'd just make this "You shouldn't use the  [`typeof` operator](https://masteringjs.io/tutorials/fundamentals/typeof) to check whether a value is an array, because `typeof` can't distinguish between arrays and objects."